cmd/evm/runner.go

```diff
@@ -236,6 +236,10 @@ Gas used:           %d
 	}
 	if tracer != nil {
 		tracer.CaptureEnd(ret, initialGas-leftOverGas, execTime, err)
+		fmt.Printf("0x%x", ret)
```

Comment:
I think the 'proper' solution would be to configure the tracer used to do the printout during `CaptureEnd` ... it's been a while since i looked at the code, though. Doing it tihs way would cause printing on stdout even if `--json` flag is used, iiuc, which would break json parsing the output

Reply:
Good suggestion! I'll check debug flag for printing output.